doc/source/examples/extended_examples/hpc/hpc_ml_ga.rst

```diff
@@ -0,0 +1,448 @@
+Genetic algorithms and PyMAPDL
+==============================
+
+This example shows how to use PyMAPDL in an HPC cluster to 
+take advantage of multiple MAPDL instances to calculate each of the
+genetic algorithm population solutions.
+To manage multiple MAPDL instances, you should use the
+:class:`~ansys.mapdl.core.pool.MapdlPool` class, which allows you
+to run multiple jobs in the background.
+
+Introduction
+============
+
+Genetic algorithms are optimization techniques inspired by the principles of natural
+selection and genetics. They are used to find solutions to complex problems by mimicking
+the process of natural selection to evolve potential solutions over successive generations.
+In a genetic algorithm, a population of candidate solutions (chromosomes) undergoes a
+series of operations, including selection, crossover, and mutation, to produce new
+generations of solutions. The fittest chromosomes, which best satisfy the problem
+constraints and objectives, are more likely to be selected for reproduction, thus
+gradually improving the overall quality of solutions over time. Genetic algorithms are
+particularly useful for solving problems where traditional methods are impractical or
+inefficient, such as optimization, search, and machine learning tasks. They find
+applications in various fields, including engineering, economics, biology, and computer science.
+
+Problem definition
+==================
+
+This example shows how to use a generic algorithm to calculate the force
+required for a double-clamped beam to deform a specific amount in its
+center.
+
+The beam model is the same as :ref:`ref_mapdl_beam`.
+It is made of ``BEAM188`` elements that span for 2.2 meters,
+and it fully clamped at both ends.
+
+The PyMAPDL beam model is defined in the ``calculate_beam()`` function as follows:
+
+.. code-block:: python
+
+    def calculate_beam(mapdl, force):
+        # Initializing
+        mapdl.clear()
+        mapdl.prep7()
+
+        # Define an I-beam
+        mapdl.et(1, "BEAM188")
+        mapdl.keyopt(1, 4, 1)  # transverse shear stress output
+
+        # Material properties
+        mapdl.mp("EX", 1, 2e7)  # N/cm2
+        mapdl.mp("PRXY", 1, 0.27)  # Poisson's ratio
+
+        # Beam properties in centimeters
+        sec_num = 1
+        mapdl.sectype(sec_num, "BEAM", "I", "ISection", 3)
+        mapdl.secoffset("CENT")
+        mapdl.secdata(15, 15, 29, 2, 2, 1)  # dimensions are in centimeters
+
+        # Setting FEM model
+        mapdl.n(1, 0, 0, 0)
+        mapdl.n(12, 110, 0, 0)
+        mapdl.n(23, 220, 0, 0)
+        mapdl.fill(1, 12, 10)
+        mapdl.fill(12, 23, 10)
+
+        for node in mapdl.mesh.nnum[:-1]:
+            mapdl.e(node, node + 1)
+
+        # Define the boundary conditions
+        # Allow movement only in the X and Z direction
+        for const in ["UX", "UY", "ROTX", "ROTZ"]:
+            mapdl.d("all", const)
+
+        # constrain just nodes 1 and 23 in the Z direction
```

Comment:
```suggestion
        # Constrain just nodes 1 and 23 in the Z direction
```
Because you tend to capitalize the initial word of your comments, all of them should be formatted like this.